test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Actions/PackageIdentityResultTests.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test
+{
+    public class PackageIdentityResultTests
+    {
+        [Fact]
+        public void AccessibleName_ReturnsMessage_CultureNeutral()
+        {
+            PackageIdentityResult result = new PackageIdentityResult("test.package", new NuGetVersion(0, 0, 1));
+
+            var msg = result.AutomationName;
+
+            Assert.Equal("test.package version 0.0.1", msg);
```

Comment:
`msg` seems like its only purpose is to preserve the result of a property which may have different values over time.  Is that true?  If no, just replace `msg` with `result.AutomationName` and delete the local `msg`.  If yes, change `var msg` to `string actualResult`.

Reply:
`result.AutomationName` doesn't change over time (see comments on the property). Thus, I will just use property in the assertion.